src/main/java/frc/robot/Constants.java

```diff
@@ -16,4 +16,7 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+    public static final class Intake{
+        public static final double VELOCITY = 0.3; // the target velocity of intake's motor(m/s)
```

Comment:
You use set power for the intake so it is in percentage and not m/s

Reply:
and change the name of the variable